index.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const path = require('path')
+
+const send = require('send')
+
+const fp = require('fastify-plugin')
+
+function fastifyStatic (fastify, opts, next) {
+  if (typeof opts.sendOptions !== 'object') {
+    return next(new Error('"sendOptions" option is required'))
+  }
+  if (typeof opts.sendOptions.root !== 'string') {
+    return next(new Error('"sendOptions.root" option is required'))
+  }
+  if (!path.isAbsolute(opts.sendOptions.root)) {
+    return next(new Error('"directory" option must be an absolute path'))
+  }
+  if (typeof opts.redirectionLink404 !== 'string') {
+    return next(new Error('"redirectionLink404" option is required'))
+  }
+  if (typeof opts.redirectionLink403 !== 'string') {
+    return next(new Error('"redirectionLink403" option is required'))
+  }
+  if (typeof opts.redirectionLink500 !== 'string') {
+    return next(new Error('"redirectionLink500" option is required'))
+  }
+
+  const root = opts.sendOptions.root
+  const redirectionLink404 = opts.redirectionLink404
+  const redirectionLink403 = opts.redirectionLink403
+  const redirectionLink500 = opts.redirectionLink500
```

Comment:
This is your decision. I don't think it should be the decision of everybody else, and it is not a widespread internet practice. See on GH: https://github.com/abcde/abcde (try whatever website you want).